### PR TITLE
added Javascript and Command line sections

### PR DESCRIPTION
**Pull request by DannyPeck**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,18 @@ Basic client for a CRUD calendar application
 Here is a link to this organizations Github Workflow Standards: [Github Workflow Standards](https://github.com/cs-club-appdev/cs-club-calendar-client/wiki/Github-Workflow-Standards)
 
 ## Development Dependencies
+### Command Line Interface (CLI)
+A command line interface is a users interface to a computers operating system.  The command line works by allowing users to entering commands into the prompt and having them be executed by the computer.  Most of the technologies we will use in this project will leverage the command line including Git, Npm, Node, Ember, Express and MongoDB.
+
+This project will focus on using the Unix command line interface.
```
Comment (technetos) [inline README.md]: This should not be optional. Working on this project, we use Unix.  Use a Mac or run Linux, be it on your machine or in a virtual machine.  No exceptions.

File: README.md [hunk 2]
```diff
@@ -24,6 +36,14 @@ GitHub is a web-based Git repository hosting service, not the version control sy
 #### Resources
 [Guides](https://guides.github.com/)
 
+### JavaScript
+This project is going to be written in JavaScript.  JavaScript is a high-level, dynamic, untyped, and interpreted programming language.  Javascript is the language of choice when working in web development.
+
```
Comment (technetos) [inline README.md]: Ember?

File: README.md (hunk 1 above)
Comment (DannyPeck) [inline README.md]: I do not want to exclude people from the project because of their OS.

Comment (technetos) [inline README.md]: I do not want to have to compensate for windows lack of support

Comment (DannyPeck) [inline README.md]: Agreed.

**Pull request merged**